app/components/press-release/form.hbs

```diff
@@ -101,3 +118,10 @@
     @onAddSources={{this.addSources}}
   />
 {{/if}}
+{{#if this.showUploadModal}}
+  <PressRelease::FilesModal
+    @fileQueueName={{this.fileQueueName}}
```

Comment:
`fileQueueName` zou ik in de files-modal.js definieren ipv mee te geven als argument